enterprise/admin.py

```diff
@@ -36,13 +37,54 @@ class EnterpriseCustomerBrandingConfigurationInline(admin.StackedInline):
     can_delete = False
 
 
+class ModelChoiceIntegerField(forms.ModelChoiceField):
+    """
+    A custom integer field for admin form, this field treats an integer field as a foreign key.
+    """
+
+    def to_python(self, value):
```

Comment:
It seems me, It will show the `ID` in EnterpriseCustomer `DetailView`, right ? but the EnterpriseCustomer `ListView` will display the `objects` instead of ID in dropdown. If YES then can we display the text in dropdown with some explanatory text or ID instead of `SAMLProviderConfig object` ?